dotnet/samples/Concepts/Plugins/CreatePluginFromOpenApiSpec_Klarna.cs

```diff
@@ -5,24 +5,24 @@
 
 namespace Plugins;
 
-public class OpenAIPlugins(ITestOutputHelper output) : BaseTest(output)
+public class CreatePluginFromOpenApiSpec_Klarna(ITestOutputHelper output) : BaseTest(output)
```

Comment:
Remember to update the readme, with the new link to here.

Reply:
Thanks, updated.